firmware/quadruna/BMS/src/app/app_tractiveSystem.c

```diff
@@ -23,14 +35,26 @@ float app_tractiveSystem_getCurrent(void)
     const float low_res_current  = io_tractiveSystem_getCurrentLowResolution();
     const float high_res_current = io_tractiveSystem_getCurrentHighResolution();
 
+    float current = 0.0f;
+
     if (IS_IN_RANGE(-HIGH_RES_MAX_CURRENT_READING, HIGH_RES_MAX_CURRENT_READING, low_res_current))
     {
-        return high_res_current;
+        current = high_res_current;
     }
     else
     {
-        return low_res_current;
+        current = low_res_current;
     }
+
+    return current;
```

Comment:
btw why did you do this?

Reply:
oh, this was in a previous commit but it "current" variable stayed when I reverted some changes.